### PR TITLE
Update whatsnew

### PR DESCRIPTION
**Pull request by stephenworsley**
Reintroduce the whatsnew latest and template removed in #4921.

### CODE REVIEW

File: docs/src/whatsnew/latest.rst.template [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+   * N/A
+
+   And finally, get in touch with us on :issue:`GitHub<new/choose>` if you have
+   any issues or feature requests for improving Iris. Enjoy!
+
+
+NOTE: section below is a template for bugfix patches
+====================================================
+    (Please remove this section when creating an initial 'dev.rst')
```
Comment (trexfeathers) [inline docs/src/whatsnew/latest.rst.template:27]:
```suggestion
    (Please remove this section when creating an initial 'latest.rst')
```

I think?

File: docs/src/whatsnew/latest.rst.template [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   :body: bg-light
+   :animate: fade-in
+
+   The patches in this release of Iris include:
+
+   #. N/A
+
+NOTE: section above is a template for bugfix patches
+====================================================
+    (Please remove this section when creating an initial 'dev.rst')
```
Comment (trexfeathers) [inline docs/src/whatsnew/latest.rst.template:44]:
```suggestion
    (Please remove this section when creating an initial 'latest.rst')
```

I think?

**Review by trexfeathers — CHANGES_REQUESTED** [review]
Thanks @stephenworsley! Couple of points below, and I think you need a reference to `latest.rst` in `index.rst`, too.

File: docs/src/whatsnew/latest.rst.template (hunk 2 above)
Comment (stephenworsley) [reply docs/src/whatsnew/latest.rst.template:44]: that seems more accurate now.

**Review by trexfeathers — APPROVED** [review]

**Pull request merged**